sources/packages/backend/workflow/src/workflow-definitions/assessment-gateway-v2.bpmn

```diff
@@ -862,9 +862,12 @@ Note: the workflow process should continue its execution as a regular reassessme
           <zeebe:calledElement processId="cra-integration-income-verification" propagateAllChildVariables="false" />
           <zeebe:ioMapping>
             <zeebe:input source="=supportingUserParentTotalIncome" target="reportedIncome" />
-            <zeebe:input source="=createdSupportingUserId" target="supportingUserId" />
-            <zeebe:input source="=[&#34;parent-1-income-verification-subprocess&#34;]" target="parentSubprocesses" />
+            <zeebe:input source="=parentSupportingUserId" target="supportingUserId" />
+            <zeebe:input source="=[&#34;parent-&#34; + string(loopCounter) + &#34;-income-verification-subprocess&#34;]" target="parentSubprocesses" />
+            <zeebe:input source="=applicationId" target="applicationId" />
+            <zeebe:input source="=programYearStartDate" target="programYearStartDate" />
             <zeebe:output source="=&#34;parent-income-verification-subprocess&#34;" target="parent_income_verification_subprocess_passthrough" />
+            <zeebe:output source="=supportingUserId" target="parentSupportingUserId" />
```

Comment:
What values will the variables `supportingUserId` and `supportingUserParentTotalIncome` have in the global scope? Similar to E2E tests, will there variables be inside an array as an object?